cloudbuild.yaml

```diff
@@ -0,0 +1,24 @@
+# See https://cloud.google.com/cloud-build/docs/build-config
+
+# this must be specified in seconds. If omitted, defaults to 600s (10 mins)
+timeout: 1200s
+# this prevents errors if you don't use both _GIT_TAG and _PULL_BASE_REF,
+# or any new substitutions added in the future.
+options:
+  substitution_option: ALLOW_LOOSE
+steps:
+  - name: "gcr.io/k8s-testimages/gcb-docker-gcloud:v20190906-745fed4"
```

Comment:
How did we pick that image and tag / when might we want to update to a different tag?

Reply:
The image is public and you check the different tags here: https://console.cloud.google.com/gcr/images/k8s-testimages/GLOBAL/gcb-docker-gcloud. 
Dockerfile: https://github.com/kubernetes/test-infra/tree/master/images/gcb-docker-gcloud.

Reply:
I'm thinking about security updates to it, maybe there's a new process we need to follow to pick those up?

Reply:
these are date-sha, so you can just upgrade to the latest at will.

security should be relatively minor ™️ as this is only the clients, not dockerd/the actual GCP systems.